browser-test/src/support/admin_questions.ts

```diff
@@ -1,563 +1,563 @@
-import { Page } from 'playwright'

```

Comment:
This file contained a mix of different line ending characters, suggesting that it probably received edits from someone on Windows with a text editor that didn't respect Unix text encoding. I re-encoded the file in UTF-8 with Unix-style line endings to make it consistent. Result is a big diff though.

Reply:
I probably should have done this in a separate PR since there is a small code diff in here 😞 .